pandas/core/frame.py

```diff
@@ -1242,6 +1242,11 @@ def to_records(self, index=True, convert_datetime64=True):
     @classmethod
     def from_items(cls, items, columns=None, orient='columns'):
         """
+        DEPRECATED: from_items is deprecated and will be removed in a
```

Comment:
we no longer use DEPRECATED (it will fail the linter), instead use ..deprecated (the sphinx directive)